components/ModHealthcareProfessionalSearchbar.vue

```diff
@@ -22,5 +24,20 @@
 </template>
 
 <script lang="ts" setup>
+import { ref, type Ref } from 'vue'
 import SVGLookingGlass from '~/assets/icons/looking-glass.svg'
+import { useModerationSearchHealthcareProfessionalsStore } from '~/stores/moderationSearchHealthcareProfessionalsStore'
+
+const id: Ref<string> = ref('')
+
+const modSearchHealthcareProfessionalStore = useModerationSearchHealthcareProfessionalsStore()
+// This code is required for implementing a feature in HTML. It's currently commented out due to ESLint rules.
+// const healthcareProfessional = computed(() => modSearchHealthcareProfessionalStore.healthcareProfessional)
```

Comment:
Is this something that should be fixed before this gets merged? We should probably avoid commenting out code a feature is dependent on due to ESLint.

Rather than commenting it out, you may consider leaving a comment for this line on your PR with the ESLint error message so reviewers can pitch in with suggestions.

Reply:
This logic will be necessary for the person that will implement the UI for this part.

The problem is if a variable is unused it cause an eslint error, so this logic I had to comment out otherwise my eslint wouldn't pass. But this code  is necessary for when UI is implemented.